spec/features/companies_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+describe "Companies" do
+
+  let(:company) { FactoryBot.create(:company) }
+  describe "index" do
+    it "should have content" do
+      visit companies_path
+      expect(page).to have_selector('h1',   text:  'All the companies from the database')
+      expect(page).to have_link('Create')
+      expect(page).to have_link('Home page' )
+    end
+  end
+
+  describe "create" do
+    it "should have content" do
+      visit new_company_path
```

Comment:
this test does not make sense. You should create a company,. Fill out the form. And test it was created